### DISCUSSION TITLE
Quarto preview in default projects -> 404 not found

### DISCUSSION THREAD

**Discussion by andrewpbray** [quarto-dev/quarto-cli#6973] [Q&A] [answered]
### Description

If I create a project of type default, then try `quarto preview` I get this:
```
(base) andrewbray@MacBookPro defaultproj % ls
Defaultproj.html	Defaultproj_files	defaultproj.Rproj
Defaultproj.qmd		_quarto.yml
(base) andrewbray@MacBookPro defaultproj % quarto preview
Terminating existing preview server....DONE

Preparing to preview

Watching files for changes
Browse at http://localhost:3585/
GET: / (404: Not Found)
```
Is this not intended to work?

If it's helpful, here's my quarto:
```
(base) andrewbray@MacBookPro ~ % quarto check

[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.1: OK
      Dart Sass version 1.55.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.3.433
      Path: /Applications/quarto/bin

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.10.9 (Conda)
      Path: /Users/andrewbray/miniconda3/bin/python
      Jupyter: (None)

      Jupyter is not available in this Python installation.
      Install with conda install jupyter

      There is an unactivated Python environment in anaconda3. Did you forget to activate it?

[✓] Checking R installation...........OK
      Version: 4.3.0
      Path: /Library/Frameworks/R.framework/Resources
      LibPaths:
        - /Library/Frameworks/R.framework/Versions/4.3-x86_64/Resources/library
      knitr: 1.43
      rmarkdown: 2.21

[✓] Checking Knitr engine render......OK
```

**Comment by mcanouil** [comment]
This is an intended behaviour as you are rendering a project and serving the project from the root which expect as any website the `index.html` file which does not exist in the project as the only file is `Defaultproj.qmd`.
If you don't specify any file, Quarto can't know which to serve and currently as no fallback if there is no `index.html`.
So:
- use `quarto preview Defaultproj.qmd`
- use `quarto preview` then navigate to `http://localhost:1234/Defaultproj.html`

**Comment by andrewpbray** [comment]
Ah, thanks for explaining that, it's helpful.

This is surprising behavior to me. I try to think of default projects as similar to websites, books, and manuscripts in that they allow batch render, conditional content, and aggregated metadata. I had thought of the point in which they differed being that they lack the navigational overlay between docs that's enabled by calling on the keys `website`, `book`, and `manuscript`. I didn't realize that `quarto preview` pulls over some behavior from the other project types.

When you run `quarto preview` in an established website, it will serve up the most recently modified file on disk instead of `index.html`, correct? That behavior doesn't seem to work in `type: default`. That is: in that same default project, I modify a qmd, save it, preview, and it still 404s (whereas a website would not, even though the website would serve up the qmd, regardless of its filename).

Thanks for your help in understanding the behavior of `quarto preview`.

**Reply by mcanouil** [reply]
I have to admit that I don't know what exactly is the file preview method for book/project, but maybe there should be the same method for default project type.

**Reply by mcanouil** [reply, marked as answer]
I opened an enhancement issue to make `quarto preview` behave similarly between project types, see:
- #6976
